pkg/endpoint/bpf.go

```diff
@@ -214,13 +214,15 @@ func (e *Endpoint) addNewRedirectsFromMap(owner regeneration.Owner, m policy.L4P
 			keysFromFilter := l4.ToKeys(direction)
```

Comment:
The `l4.ToKeys` could return `PolicyKeys` directly so you wouldn't need to cast it all the time on line 217.